enumerable.rb

```diff
@@ -0,0 +1,15 @@
+module Enumerable
+  def all?
+    each { |n| return false unless yield(n) }
+  end
+
+  def any?
+    each { |n| return true if yield(n) }
+    false
+  end
+
+  def filter
+    arr = []
+    each { |n| arr.push(n) if yield(n) }
+  end
```

Comment:
- [ ] the filter method is not returning anything at the end, please return the `arr` in order to fix this.